raffles-bulletin-of-zoology.csl

```diff
@@ -0,0 +1,661 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>Raffles Bulletin of Zoology</title>
+    <title-short>RBZ</title-short>
+    <id>http://csl.mendeley.com/styles/476141641/rbz</id>
```

Comment:
It's not validating as the id needs to be in the same style as the self-link in line 7. It also needs to match your file name.

Reply:
Thanks for the comment! I will change it soon, supposedly http://www.zotero.org/styles/raffles-bulletin-of-zoology